committee-code-of-conduct/README.md

```diff
@@ -58,16 +58,16 @@ Our [Incident Report Handling Procedures](incident-process.md) describes how we
 
 The members and their terms are as follows:
 
-### Term ends on August 23, 2025
+### Term ends on February 17, 2026
```

Comment:
Same comment as https://github.com/kubernetes/community/pull/8314/files#r1941535859, regarding this new term date.